app/case/case_details.mustache

```diff
@@ -2,38 +2,61 @@
     <p>
         <a href="{{add_borrower_url}}">Add borrower</a>
     </p>
+    {{^property}}
+        <p>
+            <a href="{{add_property_url}}">Add property</a>
+        </p>
+    {{/property}}
 </div>
 
-<div id="borrowers" class="text">
-    {{#has_borrowers}}
+{{#has_borrowers}}
+    <div id="borrowers" class="text">
+
         <h2>Borrowers</h2>
-    {{/has_borrowers}}
-    <ul>
-        {{#borrowers}}
-            <li>
-                <p>
 
-                <h3>{{full_name}}</h3>
-                <h4>Contact Information</h4>
-                <dl>
-                    <dt>Email address</dt>
-                    <dd>{{email_address}}</dd>
-                    <dt>Mobile phone number</dt>
-                    <dd>{{mobile_no}}</dd>
-                </dl>
+        <ul>
+            {{#borrowers}}
+                <li>
```

Comment:
When rendered the HTLM of <li> will come out double indented because the ```{{#borrowers}}``` wont be rendered. Doesn't need to be fixed, just remember for the future